projects/app/src/server/lib/replicache.ts

```
@@ -582,31 +578,28 @@ export const _mutate = makeDrizzleMutationHandler<typeof r.schema, Drizzle>(
-          skillId,
+          skill,
           srs: null,
           dueAt: now,
           createdAt: now,
         })
         .onConflictDoNothing();
     },
-    async reviewSkill(db, userId, { skill, rating, now }) {
-      const skillId = r.rSkillId().marshal(skill);
-
+    async reviewSkill(db, userId, { skill: skill, rating, now }) {
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)